frontend/src/Screen1.js

```diff
@@ -46,21 +33,30 @@ export const Screen1 = () => (
     </H1>
 
     <TrackPageViews />
-    <ul>
-      <ListItem>
+    <UnorderedList>
+      <Li>
         <Trans>When did it take place?</Trans>
-      </ListItem>
-      <ListItem>
+      </Li>
+      <Li>
         <Trans>In general terms, who was involved?</Trans>
-      </ListItem>
-      <ListItem>
+      </Li>
+      <Li>
         <Trans>How were you affected?</Trans>
-      </ListItem>
-    </ul>
+      </Li>
+    </UnorderedList>
 
-    <Warning>
+    <Container
+      css={css`
```

Comment:
I need to add some better documentation to these, sorry! But you should be able to do all of this block of css with https://github.com/styled-system/styled-system props! You can check out the link, or the Top Banner/Footer components to see how to use props to set there :)